superset/sql_lab.py

```diff
@@ -271,7 +272,7 @@ def execute_sql_statement(  # pylint: disable=too-many-arguments,too-many-statem
         session.commit()
         with stats_timing("sqllab.query.time_executing_query", stats_logger):
             logger.debug("Query %d: Running query: %s", query.id, sql)
-            db_engine_spec.execute(cursor, sql, async_=True)
+            db_engine_spec.execute(cursor, sql)
```

Comment:
Let's leave the `async_` for now until we understand why it's here (I think it's for Hive).

```suggestion
            db_engine_spec.execute(cursor, sql, async_=True)
```